lib/generators/suspenders/install/web_generator.rb

```diff
@@ -0,0 +1,40 @@
+require "generators/suspenders/accessibility_generator"
+require "generators/suspenders/styles_generator"
+require "generators/suspenders/advisories_generator"
+require "generators/suspenders/inline_svg_generator"
+require "generators/suspenders/factories_generator"
+require "generators/suspenders/jobs_generator"
+require "generators/suspenders/lint_generator"
+require "generators/suspenders/rake_generator"
+
+module Suspenders
+  module Generators
+    module Install
+      class WebGenerator < Rails::Generators::Base
+        include Suspenders::Generators::APIAppUnsupported
+
+        class_option :css, enum: Generators::CSS_OPTIONS
+
+        def invoke_generators
+          Suspenders::Generators::AccessibilityGenerator.new.invoke_all
+          Suspenders::Generators::StylesGenerator.new([], css: css).invoke_all
+          Suspenders::Generators::AdvisoriesGenerator.new.invoke_all
+          Suspenders::Generators::InlineSvgGenerator.new.invoke_all
+          Suspenders::Generators::FactoriesGenerator.new.invoke_all
+          Suspenders::Generators::JobsGenerator.new.invoke_all
+          Suspenders::Generators::RakeGenerator.new.invoke_all
+
+          # Needs to be invoked last, since it fixes any liting violations
+          # caused by the previous generators.
+          Suspenders::Generators::LintGenerator.new.invoke_all
```

Comment:
There's a case to be made that we could fix any violations in the source.

Reply:
Maybe instead of worrying about the linting here, we do that in the template by manually running `yarn lint` and `standardrb—fix` as the very last step.